src/libs/actions/Policy.js

```diff
@@ -43,25 +43,29 @@ function getSimplifiedEmployeeList(employeeList) {
  * Takes a full policy that is returned from the policyList and simplifies it so we are only storing
  * the pieces of data that we need to in Onyx
  *
- * @param {Object} fullPolicyOrPolicySummary
- * @param {String} fullPolicyOrPolicySummary.id
- * @param {String} fullPolicyOrPolicySummary.name
- * @param {String} fullPolicyOrPolicySummary.role
- * @param {String} fullPolicyOrPolicySummary.type
- * @param {String} fullPolicyOrPolicySummary.outputCurrency
- * @param {String} [fullPolicyOrPolicySummary.value.avatarURL]
- * @param {Object} [fullPolicyOrPolicySummary.value.employeeList]
+ * @param {Object}  fullPolicyOrPolicySummary
+ * @param {String}  fullPolicyOrPolicySummary.id
+ * @param {String}  fullPolicyOrPolicySummary.name
+ * @param {String}  fullPolicyOrPolicySummary.role
+ * @param {String}  fullPolicyOrPolicySummary.type
+ * @param {String}  fullPolicyOrPolicySummary.outputCurrency
+ * @param {String}  [fullPolicyOrPolicySummary.value.avatarURL]
+ * @param {Object}  [fullPolicyOrPolicySummary.value.employeeList]
+ * @param {Boolean} isSummary
  * @returns {Object}
  */
-function getSimplifiedPolicyObject(fullPolicyOrPolicySummary) {
+function getSimplifiedPolicyObject(fullPolicyOrPolicySummary, isSummary) {
     return {
         id: fullPolicyOrPolicySummary.id,
         name: fullPolicyOrPolicySummary.name,
         role: fullPolicyOrPolicySummary.role,
         type: fullPolicyOrPolicySummary.type,
         owner: fullPolicyOrPolicySummary.owner,
         outputCurrency: fullPolicyOrPolicySummary.outputCurrency,
-        avatarURL: fullPolicyOrPolicySummary.avatarURL || '',
+
+        // "GetFullPolicy" and "GetPolicySummaryList" returns different policy objects. If policy is retrieved by "GetFullPolicy",
+        // avatarUrl will be nested within the key "value"
+        avatarURL: (isSummary ? fullPolicyOrPolicySummary.avatarURL : lodashGet(fullPolicyOrPolicySummary, 'value.avatarURL')) || '',
```

Comment:
I don't think we need the `isSummary` parameter – we can just do this:

```suggestion
        avatarURL: fullPolicyOrPolicySummary.avatarURL || lodashGet(fullPolicyOrSummary, 'value.avatarURL', ''),
```

Reply:
hmm good call